addons/stock/models/stock_rule.py

```diff
@@ -323,6 +323,13 @@ def _log_next_activity(self, product_id, note):
                 'res_model_id': self.env.ref('product.model_product_template').id,
             })
 
+    @api.model
+    def default_get(self, fields_list):
+        res = super().default_get(fields_list)
+        if 'company_id' in fields_list and not res['company_id']:
+            res['company_id'] = self.env.company.id
+        return res
+
```

Comment:
`default_get` overrides are expected to be on the beginning on the class, like other default methods ;)
Cf [Odoo coding guidelines](https://www.odoo.com/documentation/14.0/developer/misc/other/guidelines.html#symbols-and-conventions)
![image](https://user-images.githubusercontent.com/19364378/127148301-cd41cc04-3684-4e71-9eed-6851109544b4.png)